### PR TITLE
Install python-hglib

### PR DESCRIPTION
**Pull request by s0undt3ch**
Required for the HG pillar tests

### CODE REVIEW

**Comment by rallytime** [comment]
This still looks good on my end. 

Bump @twangboy - can you review this to make sure it's good on your end?

File: git/salt.sls [hunk 1]
```diff
@@ -176,6 +176,9 @@ include:
   - python.junos-eznc
   - python.jxmlease
   {%- endif %}
+  - python.python-hglib
+  - python.python-gssapi
+  - python.pyVmomi
```
Comment (twangboy) [inline git/salt.sls]: Need to gate gssapi and pyVmomi

**Comment by s0undt3ch** [comment]
@twangboy: Please confirm if this fixes the windows issues now.

**Comment by rallytime** [comment]
Hrm...I didn't see this problem before because CentOS 7 doesn't have the same error, but the `gssapi` install is causing problems in jenkins:
```
18:11:05 ----------
18:11:05           ID: gssapi
18:11:05     Function: pip.installed
18:11:05       Result: False
18:11:05      Comment: Failed to install packages: gssapi. Error: Collecting gssapi
18:11:05                 Downloading gssapi-1.2.0.tar.gz (664kB)
18:11:05                   Complete output from command python setup.py egg_info:
18:11:05                   In distributed package, building from C files...
18:11:05                   Traceback (most recent call last):
18:11:05                     File "<string>", line 1, in <module>
18:11:05                     File "/tmp/pip-build-3IDH6s/gssapi/setup.py", line 93, in <module>
18:11:05                       raise Exception("Could not find main GSSAPI shared library.  Please "
18:11:05                   Exception: Could not find main GSSAPI shared library.  Please try setting GSSAPI_MAIN_LIB yourself or setting ENABLE_SUPPORT_DETECTION to 'false'
18:11:05                   
18:11:05                   ---------------------------------------- Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-3IDH6s/gssapi/
18:11:05      Started: 18:10:34.740457
18:11:05     Duration: 2162.281 ms
18:11:05      Changes:  
```

**Comment by rallytime** [comment]
The pyvmomi install is causing some new test failures too. I am going to revert this until this can be cleaned up since we're trying to get the nitrogen branch stabilized.

**Comment by s0undt3ch** [comment]
I'll try not to forget to create separate PRs for each

**Pull request merged**